src/inverseRobot.js

```diff
@@ -7,7 +7,17 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const invertedRobot = {};
+
+  for (const key in robot) {
+    if (Object.prototype.hasOwnProperty.call(invertedRobot, robot[key])) {
+      return null;
+    }
+
+    invertedRobot[String(robot[key])] = key;
```

Comment:
Using 'String(robot[key])' is redundant since object keys are always strings. You can directly use 'robot[key]' as the key for 'invertedRobot'.